### ISSUE TITLE
Check failure stack trace

### ISSUE THREAD

**Issue by NULL-JC** [PaddlePaddle/Paddle#24591]
如果您没有查询到相似问题，为快速解决您的提问，建立issue时请提供如下细节信息：
- 标题：简洁、精准概括您的问题，例如“Insufficient Memory xxx" ”
- 版本、环境信息：
    1）PaddlePaddle版本：1.7.2.post107 
    2）CPU：i7 7700
    3）GPU：1050ti
    4）系统环境：win10 16g内存
- 训练信息
    1）单机，单卡
    2）显存信息
    3）Operator信息

[2020-05-15 21:19:37,268] [    INFO] - Installing ernie module
[2020-05-15 21:19:37,319] [    INFO] - Module ernie already installed in C:\Users\NULL_JC\.paddlehub\modules\ernie
[2020-05-15 21:19:37,322] [    INFO] - Dataset label map = {'-1': 0, '0': 1, '1': 2}
[2020-05-15 21:19:37,354] [    INFO] - Checkpoint dir: model
[2020-05-15 21:19:43,090] [    INFO] - Load pretraining parameters from C:\Users\NULL_JC\.paddlehub\modules\ernie\assets\params.
[2020-05-15 21:19:45,322] [    INFO] - Strategy with warmup, linear decay, slanted triangle learning rate, weight decay regularization, 
E:\software\program\Python37\lib\site-packages\paddle\fluid\executor.py:811: UserWarning: There are no operators in the program to be executed. If you pass Program manually, please use fluid.program_guard to ensure the current Program is being used.
  warnings.warn(error_info)
[2020-05-15 21:19:47,613] [    INFO] - Try loading checkpoint from model\ckpt.meta
[2020-05-15 21:19:47,613] [    INFO] - PaddleHub model checkpoint not found, start from scratch...
[2020-05-15 21:19:47,733] [    INFO] - PaddleHub finetune start
*** Check failure stack trace: ***

**Comment by GT-ZhangAcer** [comment]
显存不足，调低batch size试试。如果到1仍不能运行，可能这个显卡不适合搞DL，需要一块高显存的计算卡

**Comment by iclementine** [comment]
请提供更详细的报错信息以及运行出错的代码？
